src/main/java/core/basesyntax/service/WriterServiceImpl.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.service;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.IOException;
+
+public class WriterServiceImpl implements WriterService {
+    public File writetoFile(String report) {
+        File reportFile = new File("Report.csv");

```

Comment:
the path to the file must be passed in the method parameters and the data must be written in the resource field